actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -653,10 +653,32 @@ def convert_value_to_parameters(value)
         end
       end
 
+      def permit_nested_array(elements, object)
+        permitted = []
+        
+        object.each do |nested_object|
+          case nested_object
+          when Array
+            permitted << permit_nested_array(elements, nested_object)
+          else
+            permitted << nested_object.select {|nested| elements.keys.include?(nested)}
```

Comment:
It will rise error if nested_object is scalar type.
There are nested arrays with scalar types in [GeoJSON](http://geojson.org/geojson-spec.html#id4)